client/blocks/post-actions/index.jsx

```diff
@@ -72,8 +72,8 @@ const PostActions = ( {
 						post={ post }
 						showLabel={ false }
 						commentCount={ post.discussion.comment_count }
-						onClick={ toggleComments }
-						tagName="div"
+						tagName="a"
+						link={ '/comments/' + siteDomain + '/' + post.ID }
```

Comment:
We're missing the `status` here: `/comments/all/${ siteDomain }/${ post.ID }`
See: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/comments/index.js#L31

Reply:
As @Copons noted in the example we can also use string interpolation here. 
```
link={ `/comments/all/${ siteDomain }/${ post.ID }\` }
```
which is imo more readable than string concat.